infra/gcp/ensure-staging-storage.sh

```diff
@@ -103,6 +105,13 @@ for REPO; do
     color 6 "Empowering ${WRITERS} as project viewers"
     empower_group_as_viewer "${PROJECT}" "${WRITERS}"
 
+    # Enable Release Manager Associates view access to
```

Comment:
I'd rather pull these out as special-case blocks at the end, as in ensure-prod-storage.sh